docs/src/Microphysics2M.md

```diff
@@ -1,12 +1,8 @@
 # Microphysics 2M
 
-The `Microphysics2M.jl` module provides 2-moment bulk parameterization of
-  autoconversion and accretion rates.
-Rain autoconversion defines the rate of transfer form cloud liquid water to
-  rain water due to collisions between cloud droplets.
-Rain accretion defines the rate of transfer from cloud liquid water to
-  rain water due to collisions between cloud droplets and rain drops.
-The rates are implemented after Table 1 from [Wood2005](@cite)
+The `Microphysics2M.jl` module provides 2-moment bulk parameterization of cloud microphysical processes including rain autoconversion, accretion, cloud self-collection, rain self-collection, terminal velocity and rain evaporation. Rain autoconversion defines the rate of transfer from cloud liquid water to rain water due to collisions between cloud droplets. Rain accretion defines the rate of transfer from cloud liquid water to rain water due to collisions between cloud droplets and rain drops. Cloud self-collection defines the rate of change of cloud droplets number density due to collisions between cloud droplets, and similarly, rain self-collection define the rate of change of raindrops number density due to collsions between raindrops.
```

Comment:
Thank you for adding the documentation!

I would suggest maybe to flip the order of things and start the Microphysics2M module by describing all or the components of the SB2001 scheme. And then mention the remaining autoconversion/accretion schemes. This way the logic of SB2001 would be easier to follow.  

And then maybe add a couple more sentences summarising the SB2001 derivation, i.e. how they solve the problem for small droplets first and then use the universal functions, etc. So that by scanning through the documentation the reader could get some vague idea on how the scheme is made?